CHANGES.txt

```diff
@@ -1,4 +1,6 @@
 ﻿Current
+Fixed: GITHUB-1211: Include "ignored" test count in JUnit reports (Krishnan Mahadevan)
```

Comment:
Sorry, I was not clear.
I just wanted to say, that "count" word is misleading here. I believe that there should sth like:

> Include disabled/ ignored test methods in JUnit reports (Krishnan Mahadevan)

Because this task was about adding methods, not just about including a counter.